rcore/main.c

```diff
@@ -35,20 +35,22 @@ void hardware_init(void)
     KERN_LOG("init", APP_LOG_LEVEL_INFO, "Debug Init");
     rcore_watchdog_init_early();
     KERN_LOG("init", APP_LOG_LEVEL_INFO, "Watchdog Init");
+    bluetooth_init();
+    KERN_LOG("init", APP_LOG_LEVEL_INFO, "Bluetooth Init");
     power_init();
     KERN_LOG("init", APP_LOG_LEVEL_INFO, "Power Init");
     flash_init();
     KERN_LOG("init", APP_LOG_LEVEL_INFO, "Flash Init");
-    //vibrate_init();
-//     KERN_LOG("init", APP_LOG_LEVEL_INFO, "Vibro Init");
-        display_init();
+    vibrate_init();
```

Comment:
did we fix vibration?  I thought this caused hangs?

Reply:
fixed